api/v1beta1/groupversion_info.go

```diff
@@ -26,7 +26,7 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "kmm.sigs.k8s.io", Version: "v1alpha1"}
+	GroupVersion = schema.GroupVersion{Group: "kmm.sigs.k8s.io", Version: "v1beta1"}
```

Comment:
Shouldn't we use another name for the m/s repo? I don't mind if we don't, just asking.

Reply:
I don't think so. I think it would be convenient for people to use the same CR for both vanilla Kubernetes and OCP. WDYT?

Reply:
I am for all what is simpler. Let merge that way.